pest_meta/src/ast.rs

```diff
@@ -26,33 +24,33 @@ pub enum RuleType {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq)]
-pub enum Expr {
+pub enum Expr<'i> {
     Str(String),
```

Comment:
Since the lifetime was added, it probably makes sense to make `Str(&'i str)`, `Insens(&'i str)` and `Range(&'i str, &'i str)`.

Reply:
Ahh, yes! :) I was wondering why this wasn't part of the PR which introduced lifetimes to begin with. I will put it in, though it does feel a bit out of scope for this PR.